Src/VimCore/VimSettings.fs

```diff
@@ -159,6 +159,10 @@ type internal GlobalSettings() =
             (HistoryName, "hi", SettingValue.Number(VimConstants.DefaultHistoryLength), SettingOptions.None)
             (IncrementalSearchName, "is", SettingValue.Toggle false, SettingOptions.None)
             (IgnoreCaseName,"ic", SettingValue.Toggle false, SettingOptions.None)
+            (ImeCommandName, "imc", SettingValue.Toggle false, SettingOptions.None)
+            (ImeDisableName, "imd", SettingValue.Toggle false, SettingOptions.None)
+            (ImeInsertName, "imi", SettingValue.Number 0, SettingOptions.None)
+            (ImeSearchName, "ims", SettingValue.Number -1, SettingOptions.None)

```

Comment:
Shouldn't the default for "ims" be 0?

Reply:
Not sure if it is released yet, but vim master has changed the default to -1, as discussed in this issue: https://github.com/vim/vim/issues/2071.

Reply:
I stand corrected :)